templates/registration/registration_form.html

```diff
@@ -204,6 +191,16 @@ <h3 class="panel-title">Pending Booking: Summary</h3>
 {% endblock %}
 
 {% block extrajs %}
+    <!-- client side form validation -->
+    <script src="{% static 'js/waypoints.js' %}"></script>
+    <script src="{% static 'js/jquery.validate.min.js' %}"></script>
+    <script src="{% static 'js/waypoints-sticky.js' %}"></script>
+    <script src="{% static 'js/jquery.sticky-kit.min.js' %}"></script>
+    <!-- image cropping -->
+    <link  href="https://cdn.rawgit.com/fengyuanchen/cropper/v1.0.0/dist/cropper.min.css" rel="stylesheet">
```

Comment:
@bfirsh Rawgit is in sunset phase. So this should be changed. Let's vendor it together with the other javascript?

Reply:
Good idea. Fixed. Cropper is actually already in the source tree. 🙄